### ISSUE TITLE
type 'select-checkbox' creating input + label instead of input + span and is not working with materialize

### ISSUE THREAD

**Issue by Joris-SEBIRE** [mozfet/meteor-autoform-materialize#59]
Hello,

When chosing type 'select-checkbox' in AutoForm SimpleSchema, the DOM elements generated are

```
    <p>
      <input type="checkbox" value="96C5pcgwiZw5EDmuc" label="my_label" name="my_name" id="r8Q7YrTC6i2abmq4L_96C5pcgwiZw5EDmuc">
      <label for="r8Q7YrTC6i2abmq4L_96C5pcgwiZw5EDmuc">my_value</label>
    </p>
```

when newest version of materialize (1.0.0) is expecting this type of elements to work :
```
    <p>
      <label>
        <input type="checkbox" class="filled-in" checked="checked" />
        <span>Filled in</span>
      </label>
    </p>
```

Do we have any solution to make it work ?

**Comment by mozfet** [comment]
Hmmm... seems I oversaw migrating the select-checkbox component when I migrated to Materialize 1.0.0. I do not have a workaround but will accept a pull request. I will only get time to look at it next week.

**Comment by mozfet** [comment]
Sorry, I forgot about this issue. Will look at it over the weekend.

**Comment by mozfet** [comment]
Note the fix did not include the inline version, will fix that next, see #61.

**Comment by mozfet** [comment]
See Atmosphere version 4.0.12. Also added example usage in [playground](https://github.com/mozfet/meteor-autoform-materialize-playground).

**Comment by Joris-SEBIRE** [comment]
Thanks for your work ! If you could do the same for radiobuttons and remove logs, it would be perfect :) 

![image](https://user-images.githubusercontent.com/38475496/49884793-f7dd5700-fe35-11e8-9c6a-a99692528cec.png)

**Comment by mozfet** [comment]
I created a new issue https://github.com/mozfet/meteor-autoform-materialize/issues/62 to fix radiobuttons. Will look at it tomorrow.

**Comment by mozfet** [comment]
FYI - both checkbox and radio and both inline styles have been fixed in v5.0.1
